x-pack/platform/plugins/shared/fleet/server/services/package_policy.ts

```diff
@@ -1555,6 +1655,25 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
       }
     });
 
+    // Store previous revision.
+    if (appContextService.getExperimentalFeatures().enablePackageRollback) {
+      await soClient
+        .bulkCreate<PackagePolicySOAttributes>(previousPolicyRevisionsToCreate)
```

Comment:
should we check if `previousPolicyRevisionsToCreate` or  `previousPolicyRevisionsToUpdate` is empty before calling the bulk operations?